packages/blobs/src/client.ts

```diff
@@ -8,8 +8,11 @@ import { BlobsInternalError } from './util.ts'
 
 export const SIGNED_URL_ACCEPT_HEADER = 'application/json;type=signed-url'
 
+export type Conditions = { ifNotExists?: boolean } | { ifExistsWithEtag?: string }
```

Comment:
I am finding these names a bit hard to work with, particularly `ifNotExists`
What do you think about this instead?
```
export type Conditions = { onlyIfNew?: boolean } | { onlyIfMatch?: string }
```

Reply:
I feel like I spent more time trying different options for the names of these properties than implementing the functionality itself. 😂 

Interestingly, `onlyIfNew` and `onlyIfMatch` are options I considered! I still don't love any particular combination, but I'm happy to go with those if you feel they make more sense.

Updated in https://github.com/netlify/primitives/pull/303/commits/6d6b91b0645c543a27b97447a331b6ed97889c29.